_posts/2022-03-15-kubeflow-1.5-release.md

```diff
@@ -0,0 +1,50 @@
+
+Note on K8s 1.22: Although many of the Kubeflow services (KFP, AutoML, Training Operator, KServe) have started testing with K8 1.22, v1.5 was not thoroughly tested with K8s 1.22.   Additionally, critical work is still outstanding for Kubeflow Notebooks and other central services.  Although the Community always encourages testing, K8s 1.22 is not formally supported with Kubeflow v1.5.  K8s 1.22 support is on the roadmap for Kubeflow’s next release and if you are interested in K8s 1.22 support, please add a comment on this tracking [issue](https://github.com/kubeflow/kubeflow/issues/6098).
+
+## Streamlined user experience
+
+In v1.5, the Kubeflow User Interface (UI) dashboards for Notebooks, AutoML and Volume Manager more closely match KFP’s dashboard.   These enhancements include how dashboard fields are displayed and how operations are performed.  This gives users a consistent look and feel when working in the UI.   Additionally, Kubeflow notebooks and the volume manager have implemented a new configuration template for configuring and using persistent volumes and Kubernetes storage classes.  This template makes it easier to expose more storage classes and will enable easier support of popular storage offerings, including NFS.  In addition, AutoML has also updated its SDK, CI framework and parameter settings across frameworks (goptuna, optuna, hyperopt).
+
+## Installation, tutorials and documentation
+
+The Kubeflow v1.5 release process includes improvements to Kubeflow’s installation, tutorial and documentation.  For installation, the Manifest Working Group’s [documentation] (https://github.com/kubeflow/manifests) provided the starting place for the Kubeflow [Distributions](https://www.kubeflow.org/docs/started/installing-kubeflow/), and this collaboration has increased the maturity of the code and documentation for Kubeflow’s GitOps installation pattern.  The v1.5 release process included a tracking [issue]  (https://github.com/kubeflow/website/issues/3130) for the critical web pages that are updated in each release.  Additionally, there are many Working Group documentation updates and we invite you to open issues when you find documentation that needs improvement.  Kubeflow v1.5 also includes two tutorials, which are described below:
+
+**Tutorial 1** - [E2E MNIST with Kubeflow Tutorial](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb),which provides an end-to-end test sequence i.e. start a notebook, run a pipeline, execute training, hyperparameter tuning and model serving. 
+
+**Tutorial 2** - [Deploy and run your first interfere service with KServe Tutorial](https://github.com/js-ts/website-1/blob/master/content/en/docs/external-add-ons/kserve/first_isvc_kserve.ipynb) Run your first interface service on Kubeflow 1.5 i.e. define, create and check an InferenceService, post an interfere request and receive response.  Optionally run performance tests.
```

Comment:
Update link to to https://www.kubeflow.org/docs/external-add-ons/kserve/first_isvc_kserve/

Reply:
sorry didn't mentioned this,  **Tutorial 2** link change this link to https://github.com/js-ts/website-1/blob/master/content/en/docs/external-add-ons/kserve/first_isvc_kserve.ipynb to https://www.kubeflow.org/docs/external-add-ons/kserve/first_isvc_kserve/ I mentioned this link because the docs PR wasn't merged yet but now it is can you please change, else great work /lgtm